configs/disco/README.md

```diff
@@ -0,0 +1,149 @@
+# Disco Diffusion(Google Colab)
```

Comment:
It seems that you want to provide a tutorial here. Then, I will suggest provding an outline at the beginning.

Reply:
We will offer a `jupyter` notebook contain tutorials.